tor_core/config.py

```diff
@@ -175,3 +178,6 @@ class Config:
 
 # section for gifs
 config.no_gifs = []
+
+# enables debug information for the cherrypy heartbeat server
+config.heartbeat_logging = False
```

Comment:
It's an option from necessity! :smile: Cherrypy is very verbose, so we need to be able to turn off the logging so that it doesn't clutter up everything else.